src/client/pages/signUp/SignUp.jsx

```diff
@@ -4,7 +4,9 @@ import '../../styles/signUp.css'
 
 import Header from '../../components/Header';
 
+
 import URL from '../../config'
+
 const registerEndpoint = '/user/register';
```

Comment:
Could you modify these two lines to fit the way it was done in SignIn please? Basically creating a REGISTER_URL inside of client/config.js (look at LOGIN_URL for more reference)

Reply:
I'll try my best!